sw/device/silicon_creator/lib/drivers/otbn.h

```diff
@@ -148,6 +148,20 @@ rom_error_t otbn_dmem_read(uint32_t offset_bytes, uint32_t *dest, size_t len);
  */
 void otbn_zero_dmem(void);
 
+/**
+ * Software errors fatal control
+ *
+ * Attempts to change the software_errs_fatal bit in the CTRL register. When set
+ * any software error becomes a fatal error. The bit can only be changed when
+ * the OTBN status is IDLE.
+ *
+ * @param otbn OTBN instance
+ * @param enable enable or disable whether software errors are fatal
+ * @return `kErrorOtbnNoChange` if the requested change cannot be made or
+ * `kErrorOk` otherwise.
```

Comment:
Can we use the same return status (locked or no change) for the silicon creator driver and the DIF?